app/jobs/update_sample_metadata_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UpdateSampleMetadataJob < TaskJob
+  queue_with_priority 1
+  queue_as QueueNames::SAMPLES
+
+  def perform(sample_type, user, attribute_changes = [])
+    @sample_type = sample_type
+    @user = user
+    @attribute_changes = attribute_changes
+
+    Seek::Samples::SampleMetadataUpdater.new(@sample_type, @user, @attribute_changes).update_metadata
```

Comment:
I think perhaps it needs to queue a `SampleTypeUpdateJob` after updating the metadata, to refresh the cache on which attributes are blank etc.

Reply:
I thought it already did
https://github.com/seek4science/seek/blob/e824fcfcca42450fb2ad31f78164150f4d65b546/lib/seek/templates/sample_type_template_concerns.rb#L8

https://github.com/seek4science/seek/blob/e824fcfcca42450fb2ad31f78164150f4d65b546/lib/seek/templates/sample_type_template_concerns.rb#L28-L30